templates/base_index.html

```diff
@@ -359,72 +344,40 @@ <h2 class="p-heading--1 u-no-margin--bottom">Significant enterprise savings</h2>
               <a href="/engage/cloud-economics">451 study on private/public cloud costs</a>
             </li>
           </ul>
-          <hr class="p-rule is-muted" />
-          <div class="row">
-            <div class="col-3 col-start-large-4">
-              <p class="u-no-margin--bottom">
-                <a href="/contact-us" class="js-invoke-modal">Let's talk open source&nbsp;&rsaquo;</a>
-              </p>
-            </div>
-          </div>
+          <hr class="is-muted" />
+          <p class="u-no-margin--bottom">
+            <a href="/contact-us" class="js-invoke-modal p-button is-dark">Let's talk open source&nbsp;&rsaquo;</a>
```

Comment:
The button text is showing link colours here, it should be showing normal text colours like other buttons
![Screenshot 2024-06-21 at 12 09 33 PM](https://github.com/canonical/ubuntu.com/assets/62298176/47fbafbf-d066-4dff-a967-acb022ed20b0)